app/models/installed_technology.rb

```diff
@@ -249,6 +249,18 @@ def position_relative_to_buffer_name
     "position_relative_to_buffer_#{@id}_#{type}"
   end
 
+  def valid?
+    buffer.present? || valid_profile?
```

Comment:
... never mind! `valid_profile?` will return true if the profile attribute is blank. Batteries should be fine. :blush: